UltimateMedals.as

```diff
@@ -104,9 +139,9 @@ class Record {
 		this.style = style;
 		this.hidden = false;
 	}
-	
+
 	string NameString() {
-		return medals[this.medal] + " " + this.style + this.name + "\\$z";
+		return (showCircles ? medals[this.medal] + " " : "") + this.style + this.name + "\\$z";

```

Comment:
This isn't the whole story, something will need to be done about line 314 (in the new numbering).

I actually am planning to separate medal icons into their own column soon, which would make hiding them easier (or showing only them). Which would solve that issue rather cleanly. I will probably do that before the next planned release, 1.9 (1.8 might be skipped)

Reply:
It does work however, no matter what game. https://github.com/Phlarx/tm-ultimate-medals/blob/1b0a117b7365084f130792a2e45153dab69b4a20/UltimateMedals.as#L314 seems to be Turbo-only too. However in change of #6 I can remove this addition.

Reply:
Yes, the chunk around line 314 are for only super-gold, super-silver, and super-bronze; it overlays the circle with a green outline to differentiate them form the normal medals. I expect if you unhide one of those medals, it will be obvious what's going on

Yeah, if you want to revert the icon-hiding, you can, but it's no big deal since I'll be rewriting all that logic soon anyways, so it doesn't really make a difference.

Reply:
I will then just keep it in, and you can take it out when you rewrite it. Fixed commit in a second.